buildenv/jenkins/common/build

```diff
@@ -149,13 +149,13 @@ def checkout_pullrequest() {
 def build() {
     stage('Compile') {
         timestamps {
-            sh "bash ./configure --with-freemarker-jar=$FREEMARKER --with-boot-jdk=$BOOT_JDK $EXTRA_CONFIGURE_OPTIONS"
+            sh "bash ./configure --with-freemarker-jar=$FREEMARKER --with-boot-jdk='$BOOT_JDK' $EXTRA_CONFIGURE_OPTIONS"
```

Comment:
Why quotes around one and not required for any of the others?

Reply:
The boot-jdk might contain spaces if installed in the default location "C:/Program Files".
For consistency, I added quotes to freemarker too.